goat

```diff
@@ -38,6 +38,15 @@ GOAT_PATH="${GOAT_PATH:-"$HOME/.goat"}"
 
 version=2.4.3
 
+case $(uname -s) in
+	[Mm][Ii][Nn][Gg][Ww]*)
+		LN=mingw_ln
+		;;
+	*)
+		LN=unix_ln
+		;;
+esac
```

Comment:
It's fine I guess? I'd do it differently, but the way you suggest to handle different `ln`s works for me.

Reply:
I assume eventually there might be other special cases for other OSes. I wanted to leave everything working the same by default, since I can't test any other systems (except Linux) for compatibility.

Reply:
Sure. I just prefer to use functions instead of variables. It is a matter of taste at this point. 😄